requirements/deploy.txt

```diff
@@ -397,7 +393,7 @@ unidecode==1.3.7
     # via
     #   -r requirements/pip.txt
     #   unicode-slugify
-urllib3==1.26.18
+urllib3==2.0.7
```

Comment:
I suppose this should be fine because it's a dependency pinned by other dependencies. I assume they weren't compatible before with 2.x and now they are.

We should be careful when deploying this, tho. I will make a note.